test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -438,4 +441,8 @@ def tearDown(self):
             self.du.rm(path=self.dirnames, sudo=True, force=True,
                        recursive=True)
             self.dirnames[:] = []
+        if len(self.resc_files) != 0:
```

Comment:
This can be simplified to
```suggestion
        if self.resc_files:
```